Minor changes

ReverendQ opened the pull request:
Fixed a redundant article and fixed a minor typo in the Beginner's Guide

ReverendQ committed:
Rename biome-tags.md to biome-ids-tags.md

ReverendQ committed:
Delete biome-ids.md

ReverendQ committed:
Fixed typo

"aslo" to "also"

SirLich commented:
Can you actually leave biome-tags as biome-tags? When possible, I prefer not breaking the URLs people have saved, and I think biome-tags is still a pretty sensible name overall.

ReverendQ committed:
Rename biome-ids-tags.md to biome-tags.md

ReverendQ committed:
Typo fix

"quotaitons" to "quotations"

ReverendQ committed:
Fixed link

JSON Schema link was broken, it is now fixed

ReverendQ commented:
I went ahead and made the requested change, and also fixed a couple typos. I also have in my notes that there's an additional typo in the "Bedrock modeling introduction" page, but for the life of me I cannot find it, and I didn't say *what* the typo was in my notes, so I'll ignore it for now.

ReverendQ committed:
Removed extra "/"

There was an extra "/" after one of the sentences. It doesn't seem to be part of any attempted formatting, so I'm assuming it was a typo.
